client/js/uploader.api.js

```diff
@@ -326,7 +326,7 @@
 
             this._templating.updateProgress(id, loaded, total);
 
-            if (Math.round(loaded / total * 100) === 100) {
+            if (total === 0 || Math.round(loaded / total * 100) === 100) {
```

Comment:
When files are uploading you see a progress bar, but if there's no bytes to upload then we're technically done, so we need to hide the rest of the icons